apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -396,10 +424,19 @@ struct ComposeView: View {
     private func sendLiveMessage() async {
         let typedMsg = composeState.message
         let sentMsg = truncateToWords(typedMsg)
-        if composeState.liveMessage == nil,
+        if !sentMsg.isEmpty && (composeState.liveMessage == nil || composeState.liveMessage?.sent == false),
```

Comment:
this condition cannot be correct, as if:
1. message is not empty
2. AND it is already sent
it will be sent again if this function is called

Reply:
ah, no.

Reply:
possibly it's better to swap branches making creating dummy the first branch, to make conditions non-negative and less confusing.

Reply:
There is no way to swap branches since the second branch allows to be executed with `livemessage == nil` like the first branch. If I will swap them, I have to write almost identical code to it, so nothing actually changes in terms of less confusing

Reply:
there is of course, but ok...